### ISSUE TITLE
Underscores are replaced with spaces (encode then decode)

### ISSUE THREAD

**Issue by Robert-Rendell** [One-com/rfc2047#3]
Runkit example:
https://runkit.com/robert-rendell/5f7c561e75b1e2001ad8d541

Code:

```
var rfc2047 = require("rfc2047")

var step1 = 'test_Ñ.zip'
var step2 = rfc2047.encode(step1);
var step3 = rfc2047.decode(step2);
console.log(step1);
console.log(step3);
```
Output:
```
"test_Ñ.zip"
"test Ñ.zip"
```

**Comment by papandreou** [comment]
Thanks for reporting! You're right that underscores didn't properly round-trip. Fixed in 3.0.0 🤗

**Comment by Robert-Rendell** [comment]
Thank you for fixing this so quickly!
